scripts/browserstack.py

```diff
@@ -161,58 +161,51 @@ def uploadEspressoApk(espressoApkFile):
         return None
 
 
-def getAllTestClassNames():
-    testClassNames = [
-        # Hard coded tests.
-        "com.stripe.android.TestAuthorization",
-        "com.stripe.android.TestBrowsers",
-        "com.stripe.android.TestCustomers",
-        "com.stripe.android.addresselement.AddressElementTest",
-    ]
-
-    # Add LPM tests to testClassNames.
-    lpmFileNames = os.listdir(
-        "paymentsheet-example/src/androidTest/java/com/stripe/android/lpm"
-    )
-    for fileName in lpmFileNames:
-        if not fileName.endswith(".kt"):
-            continue
-        className = fileName[:-3]
-        testClassNames.append(f"com.stripe.android.lpm.{className}")
+def runFastScandir(dir, ext):
+    subfolders, files = [], []
 
-    return testClassNames
+    for f in os.scandir(dir):
+        if f.is_dir():
+            subfolders.append(f.path)
+        if f.is_file():
+            if os.path.splitext(f.name)[1].lower() in ext:
```

Comment:
do you think you could add some comments or a helper function to make it more obvious what this is doing?